### PR TITLE
Print error details while genearting thumbnails

### LINKED ISSUE #119: Unable to resize the application icon into a 128x128 image, it will be written unchanged
Please give a more clear reason why "it will be written unchanged". On `openSUSE-Tumbleweed-XFCE-Live-x86_64-Current.iso` it seems like all icons are written unchanged. Why?

```
linux@localhost:~> /home/linux/.local/bin/appimaged 
appimaged, continuous build (commit 23a3b00), build 197 built on 2019-05-24 14:27:35 UTC
Watching /home/linux/.local/bin
Watching /home/linux/Downloads
Watching /home/linux/bin
Watching /opt
Watching /usr/local/bin
WARNING: Unable to resize the application icon into a 128x128 image, it will be written unchanged.
WARNING: No icons found at "usr/share/icons"
WARNING: Using .DirIcon as default app icon
WARNING: Unable to resize the application icon into a 256x256 image, it will be written unchanged.
WARNING: Unable to resize the application icon into a 256x256 image, it will be written unchanged.
WARNING: No icons found at "usr/share/icons"
WARNING: Using .DirIcon as default app icon
WARNING: Unable to resize the application icon into a 256x256 image, it will be written unchanged.
WARNING: Unable to resize the application icon into a 128x128 image, it will be written unchanged.
WARNING: Unable to resize the application icon into a 256x256 image, it will be written unchanged.
WARNING: Unable to resize the application icon into a 128x128 image, it will be written unchanged.
WARNING: Unable to resize the application icon into a 256x256 image, it will be written unchanged.
WARNING: Unable to resize the application icon into a 256x256 image, it will be written unchanged.
Watching /run/initramfs/isoscan/Applications
Updating desktop...
Directory '/home/linux/.local/share/mime/packages' does not exist!
Warning: update-mime-database retuned non-zero exit code:
gtk-update-icon-cache: Cache file created successfully.
Finished updating desktop in 277 milliseconds.
```

### PR DESCRIPTION
**Pull request by azubieta**
fix #119

### CODE REVIEW

File: src/libappimage/desktop_integration/Thumbnailer.cpp [hunk 1]
```diff
@@ -102,10 +103,11 @@ namespace appimage {
                 /* thumbnails are always png */
                 iconHandle.save(largeThumbnailPath.string(), "png");
                 return;
-            } catch (const IconHandleError&) {
+            } catch (const IconHandleError& error) {
                 /* we fail to resize the icon because it's in an unknown format or some other reason
                  * we just have left to write it down unchanged and hope for the best. */
-                Logger::warning("Unable to resize the application icon into a 256x256 image, "
+                Logger::warning(error.what());
```
Comment (TheAssassin) [inline src/libappimage/desktop_integration/Thumbnailer.cpp]: Would prefer one warning instead of two. Right now the messages appear unrelated to each other. I guess appending `what()` should do.

Comment (azubieta) [reply src/libappimage/desktop_integration/Thumbnailer.cpp]: Fixed, example below:
`WARNING: Unable to resize the application icon into a 128x128 image: "png resizing is not supported". It will be written unchanged.`

Comment (TheAssassin) [reply src/libappimage/desktop_integration/Thumbnailer.cpp]: Isn't this a little dangerous? Shouldn't there be some "static" text as well?

I mean, the message looks fine.

Comment (azubieta) [reply src/libappimage/desktop_integration/Thumbnailer.cpp]: The two messages were mixed, please check the new changes, the ones shown above are outdated.
